src/benchmark_runner.cc

```diff
@@ -295,6 +300,10 @@ BenchmarkRunner::IterationResults BenchmarkRunner::DoNIterations() {
   BM_VLOG(2) << "Ran in " << i.results.cpu_time_used << "/"
              << i.results.real_time_used << "\n";
 
+  // In dry run mode, we report 1 iteration even if each thread runs a singular iteration.
+  if (FLAGS_benchmark_dry_run) {
```

Comment:
i don't understand this change.

Reply:
We chatted on discord about this. I might have misunderstood your answer. When we have a benchmark that has a specified number of threads (n), a dry run would run it for (1 x n) iterations total. Would we report that as 1 or (1 x n) iterations?

Reply:
we should report whatever it does.